e2e/e2etests/test_eth_withdraw_restricted_address.go

```diff
@@ -71,5 +74,42 @@ func TestEtherWithdrawRestricted(r *runner.E2ERunner, args []string) {
 	// revert address should receive the amount
 	revertBalanceAfter, err := r.ETHZRC20.BalanceOf(&bind.CallOpts{}, revertAddress)
 	require.NoError(r, err)
-	require.EqualValues(r, new(big.Int).Add(revertBalanceBefore, amount), revertBalanceAfter)
+
+	userBalanceAfterUint := math.NewUintFromBigInt(revertBalanceAfter)
+	userBalanceBeforeUint := math.NewUintFromBigInt(revertBalanceBefore)
+	totalRevertAmount := getTotalRevertedAmount(r, cctx)
+
+	require.EqualValues(r, userBalanceAfterUint.Sub(totalRevertAmount), userBalanceBeforeUint)
+}
+
+func getTotalRevertedAmount(r *runner.E2ERunner, cctx *crosschaintypes.CrossChainTx) math.Uint {
```

Comment:
i think RefundedAmount would be better, since revert is already used terminology